libcontainerd/client_linux.go

```diff
@@ -464,7 +464,7 @@ func (clnt *client) Restore(containerID string, attachStdio StdioCallback, optio
 	cont, err := clnt.getContainerdContainer(containerID)
 	// Get its last event
 	ev, eerr := clnt.getContainerLastEvent(containerID)
-	if err != nil || cont.Status == "Stopped" {
+	if err != nil || cont.Status == "stopped" {
```

Comment:
good catch!

Could you use "github.com/containerd/containerd/runtime".Stopped instead though?